charts/victoria-metrics-distributed/values.yaml

```diff
@@ -4,91 +4,131 @@ nameOverride: "vm-distributed"
 # -- Overrides the chart's computed fullname.
 fullnameOverride: ""
 
-vmauthIngestGlobal:
-  # -- Create a vmauth as the global write entrypoint
-  enabled: true
-  # -- Override the name of the vmauth object
-  name: ""
-  # -- VMAuth spec. More options can be found [here](https://docs.victoriametrics.com/operator/api/#vmauthspec)
-  spec: {}
+# -- Global chart properties
+global: {}
 
-vmauthQueryGlobal:
-  # -- Create vmauth as the global read entrypoint
-  enabled: true
-  # -- Override the name of the vmauth object
-  name: ""
-  # -- VMAuth spec. More options can be found [here](https://docs.victoriametrics.com/operator/api/#vmauthspec)
-  spec: {}
+common:
+  vmauth:
+    spec: {}
+  vmagent:
+    # -- Common VMAgent spec, which can be overriden by each VMAgent configuration.
+    # Available parameters can be found [here](https://docs.victoriametrics.com/operator/api/index.html#vmagentspec)
+    spec:
+      remoteWriteSettings:
+        useMultiTenantMode: true
```

Comment:
It's better to not set `useMultiTenantMode=true` by default, since it degrades vmagent performance.

Reply:
it was set by default in templates, I've moved it to variables